custom.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.svg' {
+  const content: any;
+  export default content;
+}
```

Comment:
this is as defined below: https://webpack.js.org/guides/typescript/#importing-other-assets